be/src/formats/parquet/chunk_writer.cpp

```diff
@@ -51,8 +52,8 @@ Status ChunkWriter::write(Chunk* chunk) {
         ++leaf_column_idx;
     };
 
-    for (size_t i = 0; i < chunk->num_columns(); i++) {
-        auto& col = chunk->get_column_by_index(i);
+    for (size_t i = 0; i < _schema->field_count(); i++) {
```

Comment:
process struct columns

Reply:
changed to `_type_descs.size()`

In fact, we could assume `_type_descs.size() == _schema->field_count()`, which is the number of children of the root node in the schema.